### PR TITLE
Updated iw4x tutorial to add more steps for the extraction

### PR DESCRIPTION
**Pull request by Rackover**
## Pull request type

 

Please check the type of change your PR introduces:

- [x] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Other (please describe): 

## What is the current behavior?

People manage to do the first part of the tutorial consistently, then do very poorly on the last few steps to install and extract iw4x onto their game installation. I think nine out of ten people we get in iw4x support channels come because of that, or because they encountered that issue and then dropped random DLLs in their game folder and need help.

## What is the new behavior?

Tutorial is a bit more clear on this steps and explicitely says what to do and not to do about video guides and DLLs. It includes several new screenshots, which may help if the user has reading comprehension troubles.

## Other information

My english is far from perfect and some formulations / writing may be a bit weird, so by all means feel free to point out any mistake and correct my writing :)

### CODE REVIEW

**Comment by Rackover** [comment]
image

**Comment by Rackover** [comment]
image

**Comment by Rackover** [comment]
image

**Comment by Rackover** [comment]
image

File: support_iw4x_client.html [hunk 1]
```diff
@@ -71,37 +71,62 @@ <h2 class="section-title wow fadeIn" data-wow-duration="1000ms" data-wow-delay="
...
+							
+                            <li>Open the <a target=_blank href="https://dss0.cc/alterwarez/download/iw4x_full_game.torrent">full game torrent</a> in qBittorrent, making sure to match these settings.</li>
                             <img src="https://i.imgur.com/3ep6jwM.png" width="500" height="400" alt="qBittorrent Settings" class="img-fluid">
+							
                             <li>Once your torrent says <code>Seeding</code>, click on the torrent and hit the pause button.</li>
                             <img src="https://i.imgur.com/Mg3yxSn.png" width="500" height="200" alt="qBit Stop Seeding" class="img-fluid">
-                            <li>Download <a href="https://xlabs.dev/iw4x_download">IW4x</a> and unzip it's contents into your game directory.</li>
-                            <li>To start the game, double-click IW4x.exe.</li>
+							
+                            <li><b>Download the <a target=_blank href="https://dss0.cc/updater/iw4x_files.zip">IW4x</a> archive</b></li>
```
Comment (ChxseH) [inline support_iw4x_client.html:114]: Link to `https://xlabs.dev/iw4x_download` instead please.

File: support_iw4x_client.html [hunk 2]
```diff
@@ -71,37 +71,62 @@ <h2 class="section-title wow fadeIn" data-wow-duration="1000ms" data-wow-delay="
...
+							
+							<li>Once downloaded, click on the iw4x archive in your downloads to open it</li>
+							<img src="img/iw4x_tutorial_downloads.png" width="500" alt="downloads" class="img-fluid">
+							
+							<li>On the top toolbar, click <b>"Extract All"</b></li>
+							<img src="img/iw4x_tutorial_extract_all.png" width="500" alt="extract_all" class="img-fluid">
+							
+							<li>In the extraction wizard, fill in the location <b>in which you downloaded the game</b>.
+							<br><i>If you followed the tutorial closely up to this point, it should be <code>C:/Games/IW4x</code></i></li>
```
Comment (ChxseH) [inline support_iw4x_client.html:123]: Your image shows `C:\Games\IW4x` so we should match that so its not confusing.

File: support_iw4x_client.html [hunk 3]
```diff
@@ -71,37 +71,62 @@ <h2 class="section-title wow fadeIn" data-wow-duration="1000ms" data-wow-delay="
...
+							<img src="img/iw4x_tutorial_extract_path.png" width="500" alt="extract_path" class="img-fluid">
+							
+							<li>Click <b>Extract</b> and wait for the extraction to finish</li>
+							
+							<li><b style="color:lightgreen;">You're all done!</b> To start the game, double-click IW4x.exe.<br><b>Your folder should look something like this. If it does not, <span style="color:red">you missed a step in the tutorial.</span></b> Please try again carefully before requesting assistance.
+							<img src="img/iw4x_tutorial_final.png" width="500" alt="optimal_folder" class="img-fluid">
```
Comment (ChxseH) [inline support_iw4x_client.html:129]: Do we still include `iw4x-updater.exe`? That (iirc) is old and doesn't function anymore.

**Review by ChxseH — CHANGES_REQUESTED** [review]

**Pull request merged**